Matrix.SynapseInterop.Worker.FederationSender/FederationClient.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System;
+using System.Linq;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Text;
+using System.Threading.Tasks;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace Matrix.SynapseInterop.Worker.FederationSender
+{
+    class FederationClient
+    {
+        private SigningKey key;
+        private HttpClient client;
+        private string origin;
+        public FederationClient(string serverName, SigningKey key)
+        {
+            origin = serverName;
+            client = new HttpClient();
```

Comment:
lint that doesn't exist: assign variables where they are defined, not in the constructor